clamav-1.4-docker.yaml

```
@@ -0,0 +1,41 @@
+package:
+  name: clamav-1.4-docker
+  version: "1.4.2"
```

In here, we should consider using the git specific version so that it's tracked on a weekly basis at least. 
```suggestion
  version: 0.0.0_git20250131
```

---

There are no versions for the clamav-docker repo, only for the source repo of clamav. Don't know if that version your recommending is coming from some where or if it's just and example